MaterialDesignThemes.Wpf/StylizedBehaviorCollection.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Microsoft.Xaml.Behaviors;
+
+namespace MaterialDesignThemes.Wpf;
+
+public class StylizedBehaviorCollection : FreezableCollection<Behavior>
```

Comment:
Thoughts on renaming this to simply "BehaviorCollection"?

Reply:
I am good with that. As mentioned in the PR description, much of this is "stolen" from MahApps, so perhaps credit should be given where credit is due. Renaming may blur that somewhat, but I will let you decide on that.

Reply:
Yea, let's go ahead and rename the class. You raise a good point, and I think we should leave a comment at the top indicating that the source of this code was based on their work and linking to their source file.

Reply:
Renamed. Did not actually include a link-back to MahApps in this file because it is so simple, and with the rename there is not much left from the MapApps.Metro repo. Instead I added the link-back in the `BehaviorsAssist` class.